### PR TITLE
Sorting fixed

### PR DESCRIPTION
**Pull request by rhlsthrm**
## The Problem

Sorting logic was backwards and not unit tested.

## Checklist

- [ ] Test manually on `test-ui` using remote chains.
- [ ] Run load tests.
- [ ] Update documentation if needed.
- [ ] Update CHANGELOG.md.

### CODE REVIEW

File: packages/sdk/test/unit/sdkBase.spec.ts [hunk 1]
```diff
@@ -584,6 +584,32 @@ describe("NxtpSdkBase", () => {
...
+          inbox: "inbox",
+          data: { bidSignature, bid: { ...auctionBid, amountReceived: "100000" }, gasFeeInReceivingToken: "0" },
+        });
+        messageEvt.post({
+          inbox: "inbox",
+          data: { bidSignature, bid: { ...auctionBid, amountReceived: "100002" }, gasFeeInReceivingToken: "0" },
+        });
+        messageEvt.post({
+          inbox: "inbox",
+          data: { bidSignature, bid: { ...auctionBid, amountReceived: "100004" }, gasFeeInReceivingToken: "0" },
```
Comment (jakekidd) [inline packages/sdk/test/unit/sdkBase.spec.ts:605]: do we need to test gasFeeInReceivingToken being different?

Comment (jakekidd) [reply packages/sdk/test/unit/sdkBase.spec.ts:605]: p sure no but just double checking

Comment (rhlsthrm) [reply packages/sdk/test/unit/sdkBase.spec.ts:605]: No that's just a convenience thing it's already subtracted.

Comment (jakekidd) [reply packages/sdk/test/unit/sdkBase.spec.ts:605]: got it

**Pull request merged**